stratagems/gameplay/spellhold_cost.tpa

```diff
@@ -20,6 +20,8 @@ BEGIN
 		  OUTER_PATCH_SAVE string "%string%" BEGIN
 			REPLACE_TEXTUALLY "20\([, \.]?\)\(000\)" "%asking_price_thousands%\1\2"
 			REPLACE_TEXTUALLY "15\([, \.]?\)\(000\)" "%actual_price_thousands%\1\2"
+			REPLACE_TEXTUALLY "20 000" "%asking_price_thousands% 000"

```

Comment:
I think you can add this special space into regexp group and it should work. No need to separated line. Also it would be good to add some comment why there are double spaces. From editor it looks like the same character.

Reply:
I tried in the regex, but it didn't work. This special characters weren't taken into account, like the accented characters

Reply:
In separated line is it work? So add comment and it should be fine :)